src/Component/Metadata/CreateOperationInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Metadata;
+
+/**
+ * The Operation creates content.
+ *
+ * @experimental
+ */
+interface CreateOperationInterface
```

Comment:
Let's put operation methods here or in separate `OperationInterface`

Reply:
This CreateOperationInterface will be important to identify it later. On APIP, we sometimes can referred to the http method, but not in our case, cause we have GET or POST method for a create operation.

Reply:
I'm not yet convinced about the need for it if it is going to be empty. But for sure, it is better to have a marking interface instead of some method

Reply:
Could we keep this in mind, and let it as is for now?